src/cmd/ksh93/tests/attributes.sh

```diff
@@ -269,8 +269,16 @@ else
     log_error 'typeset -usi cannot be used for unsigned short'
 fi
 
+# Check ${parameter:-word} style of parameter expansion. ':' is optional and is used to check for null (empty string).
+[[ $($SHELL -c 'unset foo;typeset -Z2 foo; print ${foo-3}' 2> /dev/null) == 3 ]]  || log_error  '${foo-3} not 3 when typeset -Z2 field undefined'
```

Comment:
I know you're just copying the style of the existing tests but the stderr redirection isn't needed. And if an error is emitted it would be nice to see it. What I don't understand is why the person who wrote the original tests felt the need to redirect stderr.